packages/go/ein/ad_test.go

```diff
@@ -22,18 +22,19 @@ import (
 	"github.com/specterops/bloodhound/ein"
```

Comment:
Most of the changes here are just replacing strings with ad constants and cleaning up tests to be more concise
Actual testing of the new changes are added at the bottom of the diff